feature-libs/quote/components/summary/seller-edit/quote-summary-seller-edit.component.service.ts

```diff
@@ -49,7 +48,7 @@ export class QuoteSummarySellerEditComponentService {
   }
 
   /**
-   * Retrieves formatter according to current locale and currency
+   * Retrieves formatter according to current locale and unit
```

Comment:
Could you, please, add a dot at the end of the comment sentence and an empty line between the comment and @param / @returns? Thanks!